src/com/magento/idea/magento2uct/ui/ConfigurationDialog.java

```diff
@@ -59,6 +54,9 @@
     private JLabel modulePathError;//NOPMD
     private JLabel enableComment;//NOPMD
     private JLabel enableCommentPath;//NOPMD
+    private JCheckBox hasAdditionalPath;
```

Comment:
Please, keep labels separately from other elements. Move this one ☝️

Reply:
Thanks for your code review. I fixed. Please check.